keras2onnx/ke2onnx/upsample.py

```diff
@@ -29,6 +29,12 @@ def convert_keras_upsample(scope, operator, container, n_dims):
     else:
         raise ValueError('Unsupported dimension %s when converting Keras Upsampling layer' % n_dims)
 
+    mode = 'nearest'
+    if n_dims == 2:
```

Comment:
I suggest that we do linear mode for various n_dims in this PR. If n_dims = 1 or 3, we also need handle linear mode.

Reply:
I've made changes to remove the dimension check, to make it independent of the dimensions.